wordpress-mobile/WordPress-iOS
# Stories: Android compatibility

Fixes cross compatibility issue with Android. Prior to this change, Android stories would not open in the Stories editor.

Additional changes:
- Fixes a leak of `StoryEditor`
- Fix the mime type of uploaded files

## Testing
* Create a story with the Android application
* Open the same post from iOS
* Attempt to edit the Story (tap on the Edit button on the block)
* Story should load with flattened images.

### Story in Reader
* Open a created story in the Reader.
* Verify that the thumbnail image is displayed.

PR submission checklist:
